README.md

```diff
@@ -794,6 +794,7 @@ This repository also provides one such platforms where contributers come over an
 |[Pottery-Game](https://github.com/kunjgit/GameZone/tree/main/Games/Pottery-Game)|
  | [Ganesh_QR_Maker](https://github.com/kunjgit/GameZone/tree/main/Games/Ganesh_QR_Maker) |
 |[Wheel_of_Fortunes](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Wheel_of_Fortunes)|
+|[Samurai_Fighting_Game](https://github.com/Saipradyumnagoud/GameZone/tree/main/Games/Samurai_Fighting_Game)|
```

Comment:
change the format of the link as other games replace your name with the name of this repository

Reply:
@ishita-43 Done.